addons/web/controllers/main.py

```diff
@@ -1417,6 +1417,7 @@ class Binary(http.Controller):
         '/web/content/<string:xmlid>/<string:filename>',
         '/web/content/<int:id>',
         '/web/content/<int:id>/<string:filename>',
+        '/web/content/<int:id>-<string:unique>/<string:filename>',
```

Comment:
That's a good question. This is the type of URL that is generated when creating video attachments, and I added it since I merged the /web/video route with the /web/content route. I am not sure of how to correctly change the way those URLs are generated for the video attachments, so I preferred the simpler solution which was to add the pattern here.

How would you proceed to change the way URLs are generated for uploaded videos ?